pkgs/applications/audio/soundux/default.nix

```diff
@@ -0,0 +1,136 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, makeBinaryWrapper
+, autoPatchelfHook
+, pipewire
+, libpulseaudio
+, libappindicator
+, libstartup_notification
+, openssl
+, libwnck
+, pcre
+, util-linux
+, libselinux
+, libsepol
+, libthai
+, libdatrie
+, xorg
+, libxkbcommon
+, libepoxy
+, dbus
+, at-spi2-core
+, gnome2
+, webkitgtk
+, yt-dlp
+, ffmpeg
+, lsb-release
+}:
+
+stdenv.mkDerivation rec {
+  pname = "soundux";
+  version = "0.2.7";
+
+  src = fetchFromGitHub {
+    owner = "Soundux";
+    repo = "Soundux";
+    rev = version;
+    sha256 = "sha256-aSCsg6nJt6F+6O7UeXnvYva0vllTfsxK/cjaeOhObZY=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    makeBinaryWrapper
+    autoPatchelfHook
```

Comment:
Why do we need that if we compile it?

Reply:
I'm assuming you are referring to `makeBinaryWrapper`?  The [nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/#fun-makeWrapper) states that `makeBinaryWrapper` is usually preferred over `makeWrapper`.

Reply:
> I'm assuming you are referring to makeBinaryWrapper?

No, autoPatchelfHook

Reply:
You're right, I should not be using `autoPatchelfHook`.  I had misunderstood its purpose.